Client/Client App/Client App/SearchForm.Designer.cs

```diff
@@ -0,0 +1,219 @@
+﻿namespace Client_App
```

Comment:
I think this file was staged by mistake since it's on the client side so please unstage it